addons/mrp_repair/views/mrp_repair_views.xml

```diff
@@ -37,6 +39,13 @@
                    <field name="state" widget="statusbar" statusbar_visible="draft,confirmed,done"/>
                </header>
                <sheet string="Repairs order">
+                    <div class="oe_button_box" name="button_box">
+                        <button name="action_created_invoice"
+                            type="object"
+                            class="oe_stat_button"
+                            icon="fa-pencil-square-o" string="Invoices" attrs="{'invisible': ['|',('invoice_method','=','none'), ('state','in',('confirmed','2binvoiced','draft'))]}">
```

Comment:
The string should be Invoice (singular) because there could be only one invoice linked to the repair.